packages/turbo-repository/package.json

```diff
@@ -5,8 +5,8 @@
   "bugs": "https://github.com/vercel/turborepo/issues",
   "homepage": "https://turbo.build/repo",
   "scripts": {
-    "build": "napi build --platform -p turborepo-napi --cargo-cwd ../../ --cargo-name turborepo_napi native --js false --dts ../js/index.d.ts && mkdir -p js/dist && cp js/index.js js/dist/index.js && cp js/index.d.ts js/dist/index.d.ts",
-    "build:release": "napi build --release --platform -p turborepo-napi --cargo-cwd ../../ --cargo-name turborepo_napi native --js false",
```

Comment:
was it intentional that we, previously, didn't generate the dts file when in release mode?

Reply:
Probably not, the release process for this package is far more ad-hoc than the primary `turbo` release workflow.